create-message/src/test/java/no/unit/nva/publication/messages/create/CreateMessageHandlerTest.java

```diff
@@ -66,7 +70,7 @@ public void initialize() throws TransactionFailedException {
         messageService = new MessageService(client, Clock.systemDefaultZone());
         doiRequestService = new DoiRequestService(client, Clock.systemDefaultZone());
         environment = setupEnvironment();
-        StorageModelConstants.updateEnvironment(environment);
+        ServiceEnvironmentConstants.updateEnvironment(environment);
```

Comment:
StorageModelConstants moved to publication-commons, since they URI IDs are not stored in the database.